crates/sui-framework/src/natives/crypto.rs

```diff
@@ -27,7 +27,7 @@ pub const INVALID_RISTRETTO_GROUP_ELEMENT: u64 = 3;
 pub const INVALID_RISTRETTO_SCALAR: u64 = 4;
 pub const BULLETPROOFS_VERIFICATION_FAILED: u64 = 5;
 
-pub const BP_DOMAIN: &[u8] = b"sui";
+pub const BP_DOMAIN: &[u8] = b"mizu";
```

Comment:
we should better pick "sui". There is a concept called ["nothing up my sleeve numbers"](https://en.wikipedia.org/wiki/Nothing-up-my-sleeve_number), and "sui" will provide a better justification.